### DISCUSSION TITLE
Suggestion/Help: Allow players to "earn" additional claims

### DISCUSSION THREAD

**Discussion by MineF** [GriefPrevention/GriefPrevention#200] [Ideas]
Hello, 

I am not sure if this can be done with a different plugin or whatever. But I am thinking there should be a be a feature that works like this... 

Player joins they get an initial limit (like 1 or so) then once they play for so long get another. Then cap it to a max. 

So basically like accrued blocks  but only...  
Allow it to be set by the hours instead of per hour

**Comment by bigpresh** [comment]
I imagine it could easily be done as a GP plugin after version 20 - catching a hook for claim creation, counting how many claims they already have, and if they have too many, cancel the claim creation event and throw a message explaining it.

@RoboMWM will there be a claim creation event which plugins could cancel?

**Comment by MineF** [comment]
Say if the limit max is 8... Do you means as if they have 10 before hand they will still be able to keep those 10 until they delete them? 

I still want this to be respected to my existing players until they delete the excess claims to bring their limits down in order to create new ones.

**Comment by bigpresh** [comment]
> Say if the limit max is 8... Do you means as if they have 10 before hand they will still be able to keep those 10 until they delete them?

Yes, I mean such a plugin would catch new claim creation events, and deny them if the number of claims the user already has is equal to or higher than the number they're entitled to have.

**Comment by MineF** [comment]
Gotcha, thank you for explaining this to me. I hope GP will have these config options added as I mentioned.
